CHANGELOG.md

```diff
@@ -306,6 +306,10 @@
   inexhaustive case expressions matching on more than one subject.
   ([Surya Rose](https://github.com/GearsDatapacks))
 
+- Fixed a bug where the language server would remove the name of a constructor
+  pattern when importing an unknown module via a code action.
+  ([Surya Rose](https://github.com/GearsDatapacks))
```

Comment:
I think this code action wasn't released yet, so we can skip this changelog entry

Reply:
Ah right, yes. That's still unreleased. I'm away at the moment so I'll have to wait until I get back to remove this and fix merge conflicts